.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Expected Behavior
+
+
+## Actual Behavior
+
+
+## Steps to Reproduce the Problem
+
+  1.
+  1.
+  1.
```

Comment:
Shouldn't this be
```
1.
2.
3.
```

Reply:
Markdown spec turns this into an `<ol>` that gets auto-numbered by the browser, so 1s make it easier to reorder items.

No strong opinion on either, but this was copied from https://github.com/aerogear/aerogear-android-sdk/blob/master/.github/ISSUE_TEMPLATE.md and is already there in this format.

Reply:
Oh makes, sense, thanks.